packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -790,13 +791,35 @@ const FormModal = () => {
     }
   };
 
+  const handleConfirmClose = () => {
+    // eslint-disable-next-line no-alert
+    const confirm = window.confirm(
+      formatMessage({
+        id: 'window.confirm.close-modal.file',
+        defaultMessage: 'Are you sure? Your changes will be lost.',
+      })
+    );
```

Comment:
Are we using the default window.confirm anywhere else? I thought we always rely on our component?

Reply:
We use it in the `upload` plugin, I couldn't find "our" component, I could see use of `Prompt` but thats from `react-router-dom`.

Reply:
Ok, so we are using `window.confirm` in 3 places throughout the codebase and `ConfirmDialog` (helpler-plugin) far more often. Maybe it's worth checking what the desired behavior is according to Maeva.

Reply:
If that is a deliberate decision, I'm good with that. :)